src/prefect/runner/runner.py

```diff
@@ -693,8 +693,9 @@ async def _pause_schedules(self):
         """
         self._logger.info("Pausing all deployments...")
         for deployment_id in self._deployment_ids:
-            self._logger.debug(f"Pausing deployment '{deployment_id}'")
             await self._client.set_deployment_paused_state(deployment_id, True)
+            self._logger.debug(f"Paused deployment '{deployment_id}'")
```

Comment:
to avoid fooling ourselves into thinking we're successfully pausing things before it happens